scripts/update/Updater.php

```diff
@@ -245,5 +246,12 @@ public function update($initialVersion)
         }
 
         $this->skip('3.12.0', '3.13.1');
+
+        if ($this->isVersion('3.13.1')) {
+            OntologyUpdater::syncModels();
```

Comment:
Why sync model?

Reply:
It is not necessary? for updating the acl?

Reply:
syncModels is used to aplly modification from rdf files. USefull if the roles are defined in xml rd ffiles but not here